models/patch.go

```diff
@@ -0,0 +1,98 @@
+package models
+
+import (
+	"time"
+	"fmt"
+	"log"
+)
+
+type Patch struct {
+	Timestamp   time.Time      		 `json:"timestamp"`
+	Patch 		string 				 `json:"patch"`
+	Convo_ID    string               `json:"convo_id"`
```

Comment:
Struct fields should just be camel case.
```suggestion
	ConvoID    string               `json:"convo_id"`
```